py_zipkin/logging_helper.py

```diff
@@ -46,6 +44,7 @@ def __init__(
         log_handler,
         span_name,
         transport_handler,
+        report_root_timestamp,
```

Comment:
ps In Brave (java tracer), I call this flag traceContext.shared

```java
class TraceContext {
--snip--
  public abstract long spanId();

  /**
   * True if we are contributing to a span started by another tracer (ex on a different host).
   * Defaults to false.
   *
   * <p>When an RPC trace is client-originated, it will be sampled and the same span ID is used for
   * the server side. However, the server shouldn't set span.timestamp or duration since it didn't
   * start the span.
   */
  public abstract boolean shared();
```
https://github.com/openzipkin/brave/blob/master/brave/src/main/java/brave/propagation/TraceContext.java#L80